website/source/docs/provisioners/file.html.md

```diff
@@ -30,17 +30,16 @@ The file provisioner can upload both single files and complete directories.
 
 ## Configuration Reference
 
-The available configuration options are listed below. All elements are required.
+The available configuration options are listed below.
+
+### Required
 
 -   `source` (string) - The path to a local file or directory to upload to
     the machine. The path can be absolute or relative. If it is relative, it is
     relative to the working directory when Packer is executed. If this is a
     directory, the existence of a trailing slash is important. Read below on
     uploading directories.
 
--   `sources` (array of strings) - A list of paths or directories to upload to
-    the machine. This is similar to the `source` option.
-
```

Comment:
Why you remove sources? Why not convert source to array of string to sources and iterate over this array?
In this case packer template not grown with many files...